public/pages/Dashboard/Components/AnomaliesDistribution.tsx

```diff
@@ -29,7 +29,7 @@ import {
   //@ts-ignore
   EuiStat,
 } from '@elastic/eui';
-import { Chart, Partition, PartitionLayout } from '@elastic/charts';
+import { Chart, Partition, PartitionLayout } from '@elastic/charts/dist/index';
```

Comment:
wow, after I had pulled all of our latest changes I never ran into this issue again... you were getting this issue with 7.6.1 kibana oss?

Reply:
yes

Reply:
I use code from [here](https://github.com/elastic/kibana/tree/v7.6.1), but I run `yarn start --no-base-path --oss`. Will that make a difference?

Reply:
hmm no, I was doing the same. if you still encounter the issue i agree we should add this